### DISCUSSION TITLE
Multiple Size Supports hotkey

### DISCUSSION THREAD

**Discussion by TrojanZebra1996** [sn4k3/UVtools#831] [Ideas]
It would be incredibly practical for my workflow to be able to switch between multiple bookmarked tip diameters without having to go back to the supports menu, either a secondary button to add a support of a different size, or a hotkey to swap between 2 sizes would be very helpful.

**Comment by sn4k3** [comment]
I think I can add dropdown menu with custom profiles as well as set one as default, which also implements the idea of #604
EDIT: Implemented
